costmodel/costmodel.go

```diff
@@ -1236,6 +1295,12 @@ func (cm *CostModel) ComputeCostDataRange(cli prometheusClient.Client, clientset
 		}
 	}
 
+	networkUsageMap, err := GetNetworkUsageData(resultNetZoneRequests, resultNetRegionRequests, resultNetInternetRequests, true)
+	if err != nil {
+		klog.V(1).Infof("Unable to get Network Cost Data: %s", err.Error())
+		return nil, err
```

Comment:
Remember that returning the error here blocks costmodel execution. We probably don't want an error in the networking data fetch to halt the execution of the program, so you may just want to log here and set networkUsageMap := make(map...)

Reply:
ah, good catch. Thanks